lib/evaluate.js

```diff
@@ -0,0 +1,49 @@
+import { ethers } from 'ethers'
+
+const { BigNumber } = ethers
+
+export const evaluate = async ({
+  rounds,
+  roundIndex,
+  ieContractWithSigner,
+  logger
+}) => {
+  // Get measurements
+  const measurements = rounds[roundIndex] || []
```

Comment:
Since we are keeping all measurements in memory, I think we should handle the case where our service starts in the middle of an IE round, and we receive only some of the `MeasurementsAdded` events for this round. Unless that's something you will solve by implementing the self-healing feature? In that case, I am fine to defer that to a new pull request.

Reply:
https://github.com/filecoin-station/spark-evaluate/issues/3